Bottom: 684e7b6c521aedf5125766534de0b7c5fe7e0a2d
Top:    7939f065220b9845e51af519bd13f38fda13c0df
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-19 01:32:30 -0700

Refresh of more-reset

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index 452abe16ad7..24fc4748c41 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -336,6 +336,7 @@ public:
   }
 
   llvm::Expected<PythonObject> GetAttribute(const llvm::Twine &name) const {
+    using namespace python;
     if (!m_py_obj)
       return nullDeref();
     PyObject *obj = PyObject_GetAttrString(m_py_obj, NullTerminated(name));
